testing/docker/bookinfo.yaml

```diff
@@ -19,7 +19,6 @@ details-v1:
     - A8_SERVICE=details:v1
     - A8_REGISTER=true
     - A8_ENDPOINT_PORT=9080
-    - A8_HEALTHCHECKS=http://localhost:9080/health
   external_links:
```

Comment:
Why is this being removed? Envoy does not support health checking of its own service. It checks the health of upstream instances only.